scalahost/nsc/src/main/scala/scala/meta/internal/semantic/ConfigOps.scala

```diff
@@ -0,0 +1,12 @@
+package scala.meta.internal
+package semantic
+
+trait ConfigOps { self: DatabaseOps =>
+  object config {
```

Comment:
There is an official way to pass custom flags to compiler plugins, I think we should eventually move to that system because sys.props has problems like

- hard to use different configuration  for different projects (at least in sbt)
- jvm forking becomes problematic
- global mutable state (boo!)

Can we add a note here saying that we should ideally change to compiler -X flags in the future?